modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-webui-configuration.adoc

```diff
@@ -0,0 +1,62 @@
+= SUSE Manager {payg} WebUI setup 
+
+
+== Requirements
+
+{productname} must be setup and running on {azure}. For more information on finding and setting up {productname}, see xref:specialized-guides:public-cloud-guide/payg/azure/payg-azure-public-cloud-images.adoc[Azure Public Cloud Images].
+
+
+.The instance-flavor-check tool and {payg} schedule check 
+[NOTE]
+====
+instance-flavor-check Tool::
+The **instance-flavor-check** tool is an essential component that comes pre-installed by default. Its presence is crucial, as it determines the status of PAYG instances. When invoked, this tool identifies and categorizes the system as a PAYG instance. By default this tool is configured to run on a schedule named **update-payg-default**.
+
+update-payg-default schedule task::
+Navigate to the menu:Admin[Task Schedules] to find the **update-payg-default** task. By default, this task is set to run at 10-minute intervals. Its primary function is to monitor the presence of all running PAYG instances by executing the **instance-flavor-check tool**.
+====
+
+
+== Login to the Webui
+
+.Procedure: Login
+// Needs to be updated for azure ask Michael Calmer
+. Login to the SUSE Manager Server WebUI.
+//The username is [FILL THIS IN] the password will be the [FILL THIS IN] on {azure}.
```

Comment:
@mcalmer Can you verify the username and ID?

Reply:
@jcayouette there's still the old section https://documentation.suse.com/suma/4.3/en/suse-manager/installation-and-upgrade/pubcloud-setup.html in the docs which lists the default user name and password for the WebUI, see "Table 1. Default Administrator Account Details".